.ci/pytorch/test.sh

```diff
@@ -596,6 +596,9 @@ test_single_dynamo_benchmark() {
 
 test_inductor_micro_benchmark() {
   TEST_REPORTS_DIR=$(pwd)/test/test-reports
+  if [[ "${TEST_CONFIG}" == *cpu* ]]; then
```

Comment:
> We have to update the pre-defined numbers for CPU as well.

I was about to add this part but accidentally removed it because of a merge conflict.  Is there anything else you have in mind that we need?

Reply:
Actually the perf target in each experiment is for CUDA only, we should extend it to support multiple targets on different devices, but I think we can do it in a separate PR.